tests/Unit/PackagePrivate/Humanizer/InternationalizedHumanizerTest.php

```diff
@@ -80,4 +92,96 @@ public function simpleDateProvider(): \Generator {
 		yield [ '22nd of unknown month, 2021', new ExtDate( 2021, null, 22 ) ];
 	}
 
+	public function testSimpleDate()
+    {
+        $date = new ExtDate( 2021, 4, 3 );
+
+        $this->messageBuilder
+            ->expects($this->once())
```

Comment:
I think it is often better to use dedicated test doubles rather than the PHPUnit Mock API magic. That will also avoid the static analysis getting very mad at you :)

See point 5 in https://www.entropywins.wtf/blog/2016/05/13/5-ways-to-write-better-mocks/